### DISCUSSION TITLE
High Bandwidth Usage, Very Low Traffic & Transfer Size

### DISCUSSION THREAD

**Discussion by RickRyan26** [vercel/community#374] [Help] [answered]
After checking Chrome's Networking tab (filtered to my domain:www.standardgates.com), a full tour of my site transfers about 1MB then subsequent cached visits are closer to 50KB.

So far today, I've had 30 users and have used 25GBs of bandwidth on Vercel, which is about 850MBs per user...

Once I turn my Google Ads back on I will have closer to 7000 users per day so I'm stuck.

My techstack is mainly just SvelteKit + Firebase.

What am I missing here??

Thank you!

**Comment by samcx** [comment, marked as answer]
@RickRyan26 For the future, I recommend directing these questions towards Vercel Support (https://vercel.com/support/request). Just to confirm, are you still enquiring about your usage information?

**Reply by RickRyan26** [reply]
Support was passionately unhelpful. Turns out the total size of the build was the problem, so I moved a bunch of PDFs to external storage

**Reply by samcx** [reply]
I see. We apologize for the poor experience.

Is it possible to share with us the context of your case that was sent to Support?
